gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FieldEntity.java

```diff
@@ -52,6 +52,11 @@
   @ColumnInfo(name = "id")
   public abstract String getId();
 
+  @CopyAnnotations
+  @Nullable
+  @ColumnInfo(name = "index")
+  public abstract Integer getIndex();
```

Comment:
Can we please make this `@NonNull`? It would remove the extra complexity of having to deal with all fallbacks

Reply:
Done!